src/GTP.cpp

```diff
@@ -704,18 +717,17 @@ bool GTP::execute(GameState & game, std::string xinput) {
         } else if (winner_color == "b" || winner_color == "black") {
             who_won = FullBoard::BLACK;
         } else {
-            gtp_fail_printf(id, "syntax not understood");
+            gtp_fail_printf(id, 
+                "Unexpected winner color: %s.\n"
```

Comment:
These commands are only produced by automated tools like GTP, so there's no point to this.